examples/prefect/02-prefect-cloud.ipynb

```diff
@@ -108,6 +108,29 @@
     "!prefect auth login -t ${PREFECT_USER_TOKEN}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<hr>\n",
+    "\n",
+    "## Create a Prefect Cloud Project\n",
```

Comment:
I did not realize there was another prefect cloud test now. I wonder if that's where all the leftover flows are coming from (because the default integration-tests will just run the notebook, and finish. Doesn't know to disable schedule/delete flow at the end).

This will need an override, like the examples-cpu prefect cloud test has.

Reply:
oooo interesting. Ok yeah, this notebook was duplicated over into this place in #113

I can handle the override in the integration tests.